### ISSUE TITLE
Filter setting items cannot be saved for a long time

### ISSUE THREAD

**Issue by zhouxinghong** [smplayer-dev/smplayer#740]
**Additional context**

Filter setting items cannot be saved for a long time
![image](https://github.com/smplayer-dev/smplayer/assets/54253524/030a0705-c34e-4dda-aaf5-d6707841e1b9)

The filter setting items cannot be saved to the setting file. Every time you restart smplayer, you need to manually set it again. This is very troublesome. Please support the long-term saving of the filter setting items in the setting file to avoid the need to set it again after each startup . Thank you very much!
Windows10
22.7

**Comment by BodyDub** [comment]
Да, не возможно сохранять фильтры. При переходе на новый файл в плей листе, фильтры сбрасываются.
